controllers.js

```diff
@@ -0,0 +1,21 @@
+const { selectTopics } = require('./models');
+
+const getTopics = (req, res, next) => {
+
+    //Get endpoint
+    let endpoint = req.url.split('/');
+    endpoint = endpoint[endpoint.length - 1];
+    
+    //Get data if the endpoint is as expected or handle the error
+    if(endpoint === 'topics'){
```

Comment:
Why are you grabbing the endpoint and then validating it? Does express already handle this elsewhere (how do we get to this controller?)?

Please refer to the notes again if you're in doubt

Reply:
simplified